packages/calypso-e2e/src/lib/pages/cart-checkout-page.ts

```diff
@@ -41,7 +41,7 @@ const selectors = {
 	countryCode: `select[aria-labelledby="country-selector-label"]`,
 	postalCode: `input[id="contact-postal-code"]`,
 	submitBillingInformationButton:
-		'[data-testid="contact-form--visible"] button.checkout-button.is-status-primary',
+		'[data-testid="contact-form--visible"] checkout-submit-button button.is-status-primary',
```

Comment:
🤔 do these need a leading dot? like `.checkout-submit-button`? There's no custom element with this tag. It's strange that the e2e tests seem to pass like this but probably no purchases are made in the regular e2e tests and so this would only fail after you merged.

Reply:
That's a good catch, I'll update